lua/lualine/components/location.lua

```diff
@@ -1,6 +1,30 @@
-local function location()
-  local data = [[%3l:%-2c]]
-  return data
+local function location(args)
+  -- set when user wants to set a custom icon
+  local icons_enabled = args.icons_enabled
+
+  local icon_line_no = '' -- e0a1
+  local icon_col_no  = ''  -- e0a3
```

Comment:
I don't like this, messes up with the consistency of all components, there should be only one icon per component imo. If user wants further customization they can create their own lua function

Reply:
How about splitting them in 2  components? what common icon might line no or column no have?

Reply:
Removed icons from location .